### PR TITLE
Show how Maven 4 makes resuming after failed build easier

### PR DESCRIPTION
**Pull request by mthmulders**
Demo for https://github.com/mthmulders/snail-pace-rocket-speed-talk/issues/3.

### CODE REVIEW

File: demo-99-resume.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+mvn4 test -pl :shiro-core -am || true
+
+echo "Let's fix the test!"
+read
+
+rm ./crypto/core/src/test/groovy/org/apache/shiro/crypto/FailingTest.java
+
+echo "And follow Maven's instruction"
+echo "Note: no need to repeat the project list or mention where to resume the build!"
+
```
Comment (bmarwell) [inline demo-99-resume.sh:42]: Maybe add a read here, too, so you need to press enter to continue

File: demo-1-measure.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```
Comment (bmarwell) [inline demo-1-measure.sh:1]: File from another branch

**Review by bmarwell — CHANGES_REQUESTED** [review]
And why 99?

**Comment by mthmulders** [comment]
> And why 99?

Because I didn't know the section number yet. When the content is OK, I would renumber it.

File: demo-99-resume.sh (hunk 1 above)
Comment (mthmulders) [reply demo-99-resume.sh:43]: Yip, good idea

File: demo-1-measure.sh (hunk 2 above)
Comment (mthmulders) [reply demo-1-measure.sh:1]: Correct. I'll remove this file from the PR.

**Pull request merged**